integration-cli/docker_test_vars.go

```diff
@@ -26,28 +25,15 @@ var (
 	workingDirectory string
 )
 
-func binarySearchCommand() *exec.Cmd {
-	if runtime.GOOS == "windows" {
-		// Windows where.exe is included since Windows Server 2003. It accepts
-		// wildcards, which we use here to match the development builds binary
-		// names (such as docker-$VERSION.exe).
-		return exec.Command("where.exe", "docker*.exe")
-	}
-	return exec.Command("which", "docker")
-}
-
 func init() {
 	if dockerBin := os.Getenv("DOCKER_BINARY"); dockerBin != "" {
 		dockerBinary = dockerBin
-	} else {
-		whichCmd := binarySearchCommand()
-		out, _, err := runCommandWithOutput(whichCmd)
-		if err == nil {
-			dockerBinary = stripTrailingCharacters(out)
-		} else {
-			fmt.Printf("ERROR: couldn't resolve full path to the Docker binary (%v)", err)
-			os.Exit(1)
-		}
+	}
```

Comment:
This was actually killing me.  We were shelling out to `which` in order to ask the OS where `docker` is, when that's literally what `exec.LookPath` does (but also what `exec.Command` itself will do, which is how we could even shell out to just `which` in the first place).

I figured the point of touching `dockerBinary` at all was to get a full path to the binary, so I switched the logic a bit to run `LookPath` regardless such that `DOCKER_BINARY` could be set to something like `docker-1.4.1-dev` and have that be searched on the `PATH` as well here instead of in every single `exec.Command`.